shared/actions/devices.js

```diff
@@ -131,7 +131,7 @@ function* _deviceRevokedSaga(action: Revoke): SagaGenerator<any, any> {
       }
       yield put(setWaiting(true))
       yield call(loginDeprovisionRpcPromise, {param: {doRevoke: true, username}})
-      yield call(navigateTo([loginTab]))
+      yield put(navigateTo([loginTab]))
```

Comment:
This has happened before, right? Should we grep around to make sure this doesn't happen elsewhere